### ISSUE TITLE
Plugin broke after new update 

### ISSUE THREAD

**Issue by Yosher33** [filveith/BetterDiscord-Spotify-Lyrics-Status#30]
i don't mean to rush or anything and just wanted to know if there's an ETA on when the plugin will be fixed or you dropped support..

**Comment by aaashleyyyy** [comment]
> i don't mean to rush or anything and just wanted to know if there's an ETA on when the plugin will be fixed or you dropped support..

There are some scripts that do the same thing as this. You could use those until he updates it.

**Comment by filveith** [comment]
> i don't mean to rush or anything and just wanted to know if there's an ETA on when the plugin will be fixed or you dropped support.. 

Hi, I will try to fix it next week, but I have a lot of work at the moment, so I will see if I have the time

**Comment by YnwLNE** [comment]
> > i don't mean to rush or anything and just wanted to know if there's an ETA on when the plugin will be fixed or you dropped support..
> 
> There are some scripts that do the same thing as this. You could use those until he updates it.

examples of similar Working plugins?

**Comment by aaashleyyyy** [comment]
> > > i don't mean to rush or anything and just wanted to know if there's an ETA on when the plugin will be fixed or you dropped support..
> > 
> > 
> > There are some scripts that do the same thing as this. You could use those until he updates it.
> 
> examples of similar Working plugins?

Not plug-ins; scripts. Ovalquilter made one.

**Comment by filveith** [comment]
I started to do some research and found that discord did an update the broke some functionalities of better discord. 
For the moment better discord did not publish a fix for it, but I found a solution to fix a part of the problem. [link](https://github.com/toluschr/BetterDiscord-Animated-Status/issues/124#issuecomment-1264710900)

I will search for a way to fix the lyrics part that is still broken. 

Will keep this issue updated if I find something new
